test/index.js

```diff
@@ -20,12 +20,17 @@ import 'zone.js/dist/zone';
 import 'zone.js/dist/long-stack-trace-zone';
 import 'zone.js/dist/jasmine-patch';
 
+// Config
+import envConfig from '../env/env.config';
+
 testing.setBaseTestProviders(
     browser.TEST_BROWSER_PLATFORM_PROVIDERS,
     browser.TEST_BROWSER_APPLICATION_PROVIDERS);
 
 Object.assign(global, testing);
 
+window.ENV_PUBLIC_CONFIG = envConfig.public;
```

Comment:
Could you please clarify.

PS: I need to change this on 'global.ENV_PUBLIC_CONFIG' ?